tools/deployment/onnx2tensorrt.py

```diff
@@ -170,58 +149,92 @@ def parse_args():
         nargs='+',
         default=[58.395, 57.12, 57.375],
         help='Variance value used for preprocess input data')
```

Comment:
Add some deprecated notice in `help` to avoid users still using these arguments when they run `-h` or `--help`